minimesos/src/main/java/com/containersol/minimesos/main/Command.java

```diff
@@ -0,0 +1,10 @@
+package com.containersol.minimesos.main;
+
+public interface Command {
+
+    // TODO: this does not belong here
+    boolean isExposedHostPorts();
+    // TODO: this does not belong here
+    boolean getStartConsul();
+
```

Comment:
I created #225 for this. These methods seem to be introduced together with already removed hooks